From dc0d69b9842cee752c935daf5e43984617f179ed Mon Sep 17 00:00:00 2001
From: Qu Wenruo <wqu@suse.com>
Date: Wed, 27 Dec 2017 14:55:48 +0800
Subject: [PATCH 5/8] btrfs: qgroup-test: Fix backport error in qgroup selftest
Patch-mainline: Never, just a backport bug in selftest
References: just to make CONFIG_BTRFS_FS_RUN_SANITY_TESTS pass compile

No fs_info in btrfs_test_qgroups(), use root->fs_info instead.

Signed-off-by: Qu Wenruo <wqu@suse.com>

---
 fs/btrfs/tests/qgroup-tests.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/btrfs/tests/qgroup-tests.c b/fs/btrfs/tests/qgroup-tests.c
index 54137240cb1b..9766a4176fb2 100644
--- a/fs/btrfs/tests/qgroup-tests.c
+++ b/fs/btrfs/tests/qgroup-tests.c
@@ -470,7 +470,7 @@ int btrfs_test_qgroups(void)
 	 */
 	root->fs_info->tree_root = root;
 	root->fs_info->quota_root = root;
-	set_bit(BTRFS_FS_QUOTA_ENABLED, &fs_info->flags);
+	set_bit(BTRFS_FS_QUOTA_ENABLED, &root->fs_info->flags);
 
 	/*
 	 * Can't use bytenr 0, some things freak out
-- 
2.15.1

